### PR TITLE
Added two linux questions

### PR DESCRIPTION
**Pull request by Epik-Whale463**
## Checklist

- [x] I have read and followed the [contribution guidelines](https://github.com/freeCodeCamp/Developer_Quiz_Site/blob/main/CONTRIBUTING.md).
- [x] I have read through the [Code of Conduct](https://github.com/freeCodeCamp/Developer_Quiz_Site/blob/main/CODE_OF_CONDUCT.md) and agree to abide by the rules.
- [x] This PR is for one of the available issues and is not a PR for an issue already assigned to someone else.
- [x] My PR title has a short descriptive name so the maintainers can get an idea of what the PR is about.
- [x] I have provided a summary of my changes.

closes #XXXXX

### CODE REVIEW

File: src/data/linux-quiz.ts [hunk 1]
```diff
@@ -1,4 +1,25 @@
...
+
+  {
+    Question: "Which of the following is not a mode in the Vim editor?",
+    Answer: "Write",
+    Distractor1: "Insert",
+    Distractor2: "Visual",
+    Distractor3: "Command",
+    Explanation:
+      "In the Vim editor, there is no mode called 'Write.' The modes include Insert, Visual, and Command. 'Write' refers to a command (':w') used to save changes, not a mode.",
+    Link: "https://www.geeksforgeeks.org/entering-editing-mode-in-vim-editor/"
```
Comment (jdwilkin4) [inline src/data/linux-quiz.ts:21]: We don't accept links from external sources like geeks for geeks.
We only accept links from freeCodeCamp news publication, freeCodeCamp youtube videos, MDN or official docs.
Please change this

**Review by jdwilkin4 — COMMENTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

File: src/data/linux-quiz.ts (hunk 1 above)
Comment (Epik-Whale463) [reply src/data/linux-quiz.ts]: Thank you for pointing that out , I fixed it!

**Review by jdwilkin4 — APPROVED** [review]
Congrats on your first contribution to our repository. 🎉
We look forward to future contributions. 👍

**Pull request merged**